app/scripts/modules/core/src/config/settings.ts

```diff
@@ -72,6 +72,7 @@ export interface ISpinnakerSettings {
   bakeryDetailUrl: string;
   checkForUpdates: boolean;
   debugEnabled: boolean;
+  maxRunningExecutionsToRetrieve: 30;
```

Comment:
```suggestion
  maxRunningExecutionsToRetrieve: number;
```

Reply:
If this is typed as the default value `30`, then that's the only value typescript will allow

Reply:
got it.. Thanks.. learning. 👍